content/webapp/pages/exhibition-guide.tsx

```diff
@@ -547,7 +553,7 @@ const ExhibitionGuidePage: FC<Props> = props => {
           >
             <Layout8 shift={false}>
               <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
-                <h1 className="h1">Other exhibition guides available</h1>
+                <h2 className="h2">Other exhibition guides available</h2>
```

Comment:
Cheers for reverting, I was attempting to make it as per font size in Zeplin but then realised the ones in the design aren't mapped to any of the values we use.